report_thesis/src/sections/conclusion.tex

```diff
@@ -1 +1,32 @@
-\section{Conclusion}\label{sec:conclusion}
\ No newline at end of file
+\section{Conclusion}\label{sec:conclusion}
+This thesis set out to advance the analysis of \gls{libs} data for predicting major oxide compositions in geological samples.
+By integrating sophisticated machine learning techniques and ensemble regression models, we aimed to tackle the substantial challenges posed by the high-dimensional, nonlinear nature of \gls{libs} data.
+
+Our research confronted and addressed critical challenges, including the complexities of high dimensionality, non-linearity, multicollinearity, and the limited availability of data.
+These issues traditionally hinder the accurate prediction of major oxides from spectral data, necessitating the development of robust and adaptive computational methodologies.
+
+Throughout our study, we systematically explored a diverse range of machine learning models, categorized into ensemble learning models, linear and regularization models, and neural network models.
+By implementing a rigorous evaluation framework, we identified the strengths and limitations of each model within the context of \gls{libs} data analysis.
+
+Normalization and transformation techniques played a crucial role in our approach.
+We investigated and employed various methods such as Z-score normalization, max absolute scaling, min-max normalization, robust scaling, Norm 3, power transformation, and quantile transformation.
+These techniques were vital for standardizing the data, managing different scales, and ultimately enhancing the performance of our models.
+
+Dimensionality reduction methods like \gls{pca} and \gls{kernel-pca} seemed promising in addressing the high dimensionality of the spectral data, but did not conclusive results regarding their efficacy.
+
+One of the key innovations in our approach was the use of stacked generalization.
```

Comment:
Since stacked generalization has been used before but not on LIBS?
```suggestion
One of the key innovations in our approach was the application of stacked generalization in the context of \gls{libs} data.
```